### ISSUE TITLE
Variable binding is wrong on if-let

### ISSUE THREAD

**Issue by lucascs** [cursive-ide/cursive#500]
If I write:

``` clj
(if-let [a (something)]
  (do-something a)
  (do-a-different-thing a))
```

`a` is not available on the `do-a-different-thing` branch (else), only on the `do-something` (then).

Cursive should mark the second usage of `a` as an unbound var.

**Comment by danielcompton** [comment]
This is still present in 0.1.58.

**Comment by cursive-ide** [comment]
Fix released in 1.13.1-eap4 on 2023-09-19.

**Comment by cursive-ide** [comment]
Fix released in 1.13.1 on 2023-12-07.
